Extracion of crypto-pgp and making crypto work on FIPS (#6241)
apache/camel-quarkus: extensions/crypto-pgp/runtime/src/main/doc/limitations.adoc

=== FIPS

It may not be possible to run `crypto` and `crypto-pgp` extensions together on FIPS enabled system.

> For example if `crypro` uses `BCFIPS` provider and `crypto-pgp` uses `BC` provider, it is no possible to have bot providers on one classpath.

Reviewer: ```suggestion
For example if `crypto` uses `BCFIPS` provider and `crypto-pgp` uses `BC` provider, it is no possible to have both providers on the same classpath.
```

Author: fixed
